ui/src/views/mint/nft-card/nft-card.tsx

```diff
@@ -10,6 +10,7 @@ type NftCardProps = {
   buttonText: string;
   leftIconButton?: React.ReactNode;
   onClick: () => void;
+  isLoading: boolean;

```

Comment:
if we're adding this to the component, I think we'll need to modify the NftMinted component